api/v1alpha1/hostedcluster_types.go

```diff
@@ -1265,6 +1271,9 @@ const (
 	InsufficientClusterCapabilitiesReason = "InsufficientClusterCapabilities"
 
 	OIDCConfigurationInvalidReason = "OIDCConfigurationInvalid"
+
+	ValidReleaseImageUnsupportedVersion = "ValidReleaseImageUnsupportedVersion"
```

Comment:
can we please follow naming convention as above i.e `ValidReleaseImageUnsupportedVersionReason`
`ValidReleaseImageChangeUnsupportedReason`

Also should the value be `UnsupportedVersion and `ChangeUnsupported`? since validReleaseImage prefix is implicit in the condition type?